### DISCUSSION TITLE
14.3.0 to require all apps to be React 19 minimum?!

### DISCUSSION THREAD

**Discussion by fungilation** [vercel/next.js#65929] [Polls]
I assume Next isn't so callous and this is only a canary change, not rolling out in 14.3 release. But I want to make sure.

Quoting myself on https://github.com/vercel/next.js/pull/65806, which was merged after next 14.3.0-canary.68, the last working canary version working for my app:

> Please revert. This forced upgrade of react, react-dom to 19 when other libs aren't ready is breaking libs and apps. Mine for example.
> 
> I'm on blitz 2.0.9, next 14.3.0-canary.70
> 
> blitz/next dev --turbo:
> <img width="1506" alt="image" src="https://github.com/vercel/next.js/assets/3803466/25ada75d-a588-4bb7-bb65-c31733d09b23">
> 
> turbo build:
> <img width="1506" alt="image" src="https://github.com/vercel/next.js/assets/3803466/0c189e5e-4ff8-453a-90fc-0a7f6ecebf88">
> 
> I revert to latest working canary of next 14.3.0-canary.68 and above errors go away. When I can stay on react 18.2 in my package.json

**Comment by laneme** [comment]
Welp. I hope they're just testing stuff. Canary afterall

<img src="https://github.com/vercel/next.js/assets/26094878/5a4c0e1c-ffbf-458b-8927-01d1eeffd7a7" width="200">

**Comment by feedthejim** [comment]
We will do a 15 release soon, requiring React 19 as the standard version. However I'm happy to also do a backport of important fixes if you're stuck with React 18 for the foreseeable future.

**Reply by fungilation** [reply]
Not trying to impose, but this is 14.3.0-canary. Not 15-canary. Can the approach be the other way around instead of selective backporting? That all non-React 19 commits be cherry picked for 14.3, while everything including React 19 go to 15?

**Comment by max-mayorov** [comment]
I wonder if 14.3.0 will be released or it is abandoned in favor of 15.0.0?
